experimenter/experimenter/features/manifests/apps.yaml

```diff
@@ -5,16 +5,16 @@
 fenix:
   repo: @mozilla-mobile/firefox-android
   fml_path: fenix/app/nimbus.fml.yaml
-  branch_mapping: "v{version}"
+  branch_mapping: "releases_v{version}"
```

Comment:
I just realized that they are using tags for each of the dot releases (e.g. ) and `releases_v` syntax for the major releases (e.g. https://github.com/mozilla-mobile/firefox-android/tree/releases_v107)

Reply:
also, focus is using this syntax for tags: 
<img width="436" alt="image" src="https://github.com/mozilla/experimenter/assets/43795363/f208e2d1-4efa-4ce0-9de1-50d65c4c783d">